### PR TITLE
Unraid Branding

### PR DESCRIPTION
**Pull request by jlre249**
## Summary
Intended to close issue #11 . 
Logos will be fixed in follow on issue #13 .

This merge request deals purely with UI theming. It makes no effort to update the contents or functionality of the UI.

## Main Window
Screenshot (81)

## Storage Selector Pop-Up
| | |
| --- | --- |
| Screenshot (82) | Screenshot (83) |

## OS Selector Pop-Up
| | |
| --- | --- |
Screenshot (84) | Screenshot (85)

## Raspberry Pi Device Selector Pop-Up
| | |
| --- | --- |
Screenshot (86) | Screenshot (87)

## Use Custom OS Settings Pop-Up
Screenshot (88)

## Custom OS Settings Entry Window
| | | |
| --- | --- | --- |
Screenshot (89) | Screenshot (90) | Screenshot (91)

## Erase Existing Data Pop-Up
Screenshot (92)

## Language Selector Panel
| | |
| --- | --- |
Screenshot (94) | Screenshot (95)

## Progress Bar on Main Window
Screenshot (96)

### CODE REVIEW

**Review by elibosley — APPROVED** [review]
This looks great - nice job! I have a couple comments but if they seem like too large of a lift, feel free to merge away.

File: src/MsgPopup.qml [hunk 1]
```diff
@@ -26,18 +26,23 @@ Popup {
...
+    
+    background: Rectangle {
+        color: "#1C1B1B"
+        border.color: "#2B2A29"
+    }
 
     // background of title
     Rectangle {
-        color: "#f5f5f5"
+        color: "#2B2A29"
```
[the remark this answers is not in the document]
Comment (jlre249) [reply src/MsgPopup.qml]: According to the internet, the way to do constants in qml is via a singleton object with readonly member variables. My latest commit adds qmlcomponents/UnColors.qml to define this object with the Unraid colors I used (dark gray, medium gray, orange) and qmlcomponents/qmldir to declare the instance of this object. The RGB colors in the qml files have now been replaced with "white", "UnColors.darkGray", "UnColors.mediumGray", and "UnColors.orange".

File: src/OptionsPopup.qml [hunk 2]
```diff
@@ -132,7 +137,8 @@ Window {
 
                                     Text {
                                         text: qsTr("Username:")
-                                        color: parent.enabled ? (fieldUserName.indicateError ? "red" : "black") : "grey"
+                                        opacity: parent.enabled ? 1.0 : 0.3
+                                        color: fieldUserName.indicateError ? "red" : "#FFFFFF"
```
[the remark this answers is not in the document]
Comment (jlre249) [reply src/OptionsPopup.qml]: Same as https://github.com/unraid/usb-creator-next/pull/15#discussion_r1529515337

File: src/UseSavedSettingsPopup.qml [hunk 3]
```diff
@@ -29,15 +34,15 @@ Popup {
...
+        color: "#2B2A29"
         anchors.right: parent.right
         anchors.top: parent.top
         height: 35
         width: parent.width
     }
     // line under title
     Rectangle {
-        color: "#afafaf"
+        color: "#2B2A29"
```
[the remark this answers is not in the document]
Comment (jlre249) [reply src/UseSavedSettingsPopup.qml]: Same as https://github.com/unraid/usb-creator-next/pull/15#discussion_r1529515337

File: src/MsgPopup.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (elibosley) [reply src/MsgPopup.qml]: Love it. Thanks for swapping this out!

**Comment by elibosley** [comment]
@jlre249 feel free to merge when you're ready!

**Pull request merged**